juego.py

```diff
@@ -394,18 +395,22 @@ def run(self):
 
         self.fondo = cargar_imagen('data/1.jpg')
         self.screen.blit(self.fondo, (0, 0))
-
+        play_again = 0
         pygame.display.flip()
-        while self.running:
+        while self.running or play_again == 1:
+            play_again = 0
             level = self.main()
-            self.play(level)
+            score, puntuacionalta = self.play(level)
+            play_again = self.game_over(score, puntuacionalta)
 
     def game_over(self, score, high_score):
         sonido_menu = load_sound("menu.ogg")
         gameover = self.fuente_130.render("GAME OVER!!", True, (255, 0, 0))
         quit = self.fuente_130.render("EXIT", True, (0, 0, 0))
+        play_again = self.fuente_130.render("play again", True, (0, 0, 0))
```

Comment:
Would be nice if "play again" was all caps.

Reply:
> Would be nice if "play again" was all caps.

Sure, I missed it (but the font isn't case sensitive, everything reflects to be in caps when rendered)

> What branch is [87b6f24](https://github.com/sugarlabs/jamath-activity/commit/87b6f24772547ef344ae0bd9214394ad1c48a2ba) a rebase of?
> 
> Reviewed, not tested.

master (before the merge of #21). In next commit, I'll pull these changes.

Reply:
> Sure, I missed it (but the font isn't case sensitive, everything reflects to be in caps when rendered)

Oh okay.